src/main/java/io/in_toto/keys/RSAKey.java

```diff
@@ -1,40 +1,32 @@
 package io.in_toto.keys;
 
-import java.lang.System;
-
 import io.in_toto.lib.JSONEncoder;
 
 import java.io.IOException;
 import java.io.FileReader;
 import java.io.FileWriter;
-import java.io.FileNotFoundException;
 import java.io.Reader;
 import java.io.StringWriter;
 import java.io.Writer;
 
-import java.util.ArrayList;
 import java.util.HashMap;
 
+import org.bouncycastle.util.encoders.Hex;
+
 import org.bouncycastle.openssl.PEMKeyPair;
 import org.bouncycastle.openssl.PEMParser;
 import org.bouncycastle.openssl.MiscPEMGenerator;
 import org.bouncycastle.openssl.jcajce.JcaPEMWriter;
-
 import org.bouncycastle.asn1.x509.SubjectPublicKeyInfo;
-
-import org.bouncycastle.crypto.digests.SHA256Digest;
-import org.bouncycastle.util.encoders.Hex;
-
+import org.bouncycastle.crypto.Signer;
 import org.bouncycastle.crypto.util.PrivateKeyFactory;
 import org.bouncycastle.crypto.util.PublicKeyFactory;
 import org.bouncycastle.crypto.params.AsymmetricKeyParameter;
-
-import org.bouncycastle.crypto.signers.RSADigestSigner;
+import org.bouncycastle.crypto.digests.SHA256Digest;
```

Comment:
Why do we need this import?

Reply:
These imports are necessary for your base implementation. I just changed the order.

Reply:
These imports are necessary for your base implementation. I just changed the order.

Reply:
the diff here says that you removed RSADigestSigner and added SHA256Digest, not that you changed the order

Reply:
I removed RSADigestSigner because it was not used in the code. For SHA256Digest, it was part of your code from the first place.